### PR TITLE
REL: 2021.2

### PR DESCRIPTION
**Pull request by andrewsanchez**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ function getQIIME2Channel(buildTarget: string) {
 
     case 'release':
     default:
-      return 'qiime2/label/r2020.11'
+      return 'qiime2/label/r2021.2'
```
Comment (thermokarst) [inline src/main.ts:19]: The "staging" case just above this needs to be updated, too (to 2021.4) - thanks!

**Review by thermokarst — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 1 above)
Comment (thermokarst) [reply src/main.ts:19]: Great, thanks! Unfortunately you'll have to rerender the output, using `npm run all`.

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "@actions/artifact": "^0.2.0",
     "@actions/glob": "^0.1.0",
+    "@types/node": "^14.14.31",
```
Comment (thermokarst) [inline package.json:25]: Where did this come from?

Comment (thermokarst) [reply package.json:25]: Its not a problem, that its added, I just want to be sure I understand why it was added, before merging.

**Comment by andrewsanchez** [comment]
@thermokarst, after a simple re-run of the workflow, it looks like the check passed without any mention of issues related to the change from lockfile version 1 to 2.  I'll go ahead and assign to you for review.

File: package.json (hunk 2 above)
Comment (thermokarst) [reply package.json:25]: For any folks just passing by, @andrewsanchez & I connected offline - sounds like he wrote his responses as a PR review, and didn't submit until after my second post. Interestingly enough, GH still maintains the insertion order of the comments, even though you'll see that the timestamps don't match up at all. TIL!

**Comment by andrewsanchez** [comment]
I got a notification about this regarding a "mention," but I don't actually see any new activity here 🤔 .

**Comment by thermokarst** [comment]
> I got a notification about this regarding a "mention," but I don't actually see any new activity here 🤔 .

https://github.com/qiime2/action-library-packaging/pull/13#discussion_r585116130

Screen Shot 2021-03-01 at 4 54 46 PM

**Comment by andrewsanchez** [comment]
Interesting.  I wonder if I accidentally resolved the conversation before seeing your coment.

**Comment by thermokarst** [comment]
> I wonder if I accidentally resolved the conversation before seeing your coment.

It was already marked as "resolved" at the time that I wrote and submitted the comment.

**Review by thermokarst — APPROVED** [review]

**Pull request merged**